src/App.tsx

```diff
@@ -1,13 +1,13 @@
 import React, { useState } from 'react';
 import logo from './logo.svg';
-import './App.css';
 import Navbar from './components/Navbar';
 import InputBasic from './components/inputBasic';
+import './styles/styles.scss';
 
 function App() {
 	const [number, setNumber] = useState(0);
 	const [Value, setValue] = useState('');
-	function val(e: any) {
+	function val(e: React.ChangeEvent<HTMLInputElement>) {
 		setValue(e.target.value);
 		console.log(Value);
```

Comment:
Esse console.log está sendo utilizado para algo importante? Se não, pode remover daqui, pfvr.

Reply:
Faz um favor então. 
Sobe um comentário em cima da função APP de `TODO: Remove in the Future`